integrasjonspunkt/pom.xml

```diff
@@ -352,6 +352,12 @@
             <artifactId>spring-boot-starter-thymeleaf</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>bootstrap</artifactId>
```

Comment:
Burde helst ha dependency managment for denne i hoved-pomen, så vi slipper å ha versjonnummeret i denne pomen.